### PR TITLE
Publish the Combine Scripts extension.

### PR DESCRIPTION
**Pull request by CobusKruger**
Publish the Combine Scripts extension to the gallery. This is a publicly contributed extension.

### CODE REVIEW

File: extensions/combine-scripts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (llali) [inline extensions/combine-scripts]: please remove this file. you only need to add the extension to the gallery

File: extensionsGallery.json [hunk 2]
```diff
@@ -480,7 +480,57 @@
...
+                    "versions": [
+                      {
+                        "version": "1.1.0",
+                        "lastUpdated": "7/13/2018",
+                        "assetUri": "",
+                        "fallbackAssetUri": "fallbackAssetUri",
+                        "files": [
+                        {
+                          "assetType": "Microsoft.SQLOps.DownloadPage",
+                          "source": "https://github.com/BateleurIO/sqlops-combine-scripts/releases/tag/1.1.0"
```
Comment (llali) [inline extensionsGallery.json]: this url is wrong. I think this is the correct one? [https://github.com/BateleurIO/sqlops-combine-scripts/releases/tag/1.1](https://github.com/BateleurIO/sqlops-combine-scripts/releases/tag/1.1)

File: extensionsGallery.json [hunk 3]
```diff
@@ -480,7 +480,57 @@
...
+                        },
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.Links.Source",
+                          "source": "https://github.com/BateleurIO/sqlops-combine-scripts"
```
Comment (llali) [inline extensionsGallery.json]: please add this key to the properties bellow:

"properties": [{"key":"Microsoft.VisualStudio.Services.Links.Source"},...]

File: extensionsGallery.json [hunk 4]
```diff
@@ -480,7 +480,57 @@
...
+                       },
+                       {
+                          "assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+                          "source": "https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/icons/main-icon.png"
+                        },
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.Content.Details",
+                         "source": "https://github.com/BateleurIO/sqlops-combine-scripts/blob/master/README.md"
```
Comment (llali) [inline extensionsGallery.json]: change this url to "[https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/README.md](https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/README.md)"

File: extensionsGallery.json [hunk 5]
```diff
@@ -480,7 +480,57 @@
...
+                        },
+                        {
+                          "assetType": "Microsoft.VisualStudio.Code.Manifest",
+                          "source": "https://github.com/BateleurIO/sqlops-combine-scripts/blob/master/package.json>"
```
Comment (llali) [inline extensionsGallery.json]: this is invalid. please change to "[https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/package.json](https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/package.json)" make sure to remove the extra">" at the end

File: extensionsGallery.json [hunk 6]
```diff
@@ -480,7 +480,57 @@
...
+                        },
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.Content.License",
+                          "source": "https://github.com/BateleurIO/sqlops-combine-scripts/blob/master/LICENSE>"
```
Comment (llali) [inline extensionsGallery.json]: same as above. change to "[https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/LICENSE](https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/LICENSE)"

**Comment by CobusKruger** [comment]
@llali Thank you for your feedback, I'll get them sorted tonight. I see there was a subsequent pull request for another new extension, also using `extensionId` of 20. Should I change mine?

**Comment by CobusKruger** [comment]
@llali That's the lot of them. I think please leave the preview tag for the moment. I have a change or two that I should push soon.

**Comment by kburtram** [comment]
@llali is this PR ready now?  If so, I'll publish this extension tonight for tomorrow's July release.

**Comment by llali** [comment]
@kburtram I already published version1.1 of this extension but I have to update it to 1.2

**Pull request merged**